cmd/config/internal/commands/cmdcreatesubstitution.go

```diff
@@ -17,20 +18,19 @@ import (
 func NewCreateSubstitutionRunner(parent string) *CreateSubstitutionRunner {
 	r := &CreateSubstitutionRunner{}
 	cs := &cobra.Command{
-		Use:     "create-subst DIR NAME VALUE",
-		Args:    cobra.ExactArgs(3),
+		Use:     "create-subst DIR NAME",
+		Args:    cobra.ExactArgs(2),
 		PreRunE: r.preRunE,
 		RunE:    r.runE,
 	}
 	cs.Flags().StringVar(&r.CreateSubstitution.FieldName, "field", "",
-		"name of the field to set -- e.g. --field port")
+		"name of the field to set -- e.g. --field image")
+	cs.Flags().StringVar(&r.CreateSubstitution.FieldValue, "field-value", "",
+		"value of the field to create substitution for -- e.g. --field-value nginx:0.1.0")
 	cs.Flags().StringVar(&r.CreateSubstitution.Pattern, "pattern", "",
-		"substitution pattern")
-	cs.Flags().StringSliceVar(&r.Values, "value", []string{""},
-		"substitution values for the pattern.  format is PATTERN_MARKER=SETTER_NAME"+
-			"where PATTERN_MARKER is the pattern substring to replace, and SETTER_NAME is the"+
-			"setter from which to take the replacement value.")
+		"substitution pattern -- e.g. --pattern ${my-image-setter}:${my-tag-setter}")
```

Comment:
Seems this help example cannot be taken literally;
the shell will attempt to expand `${my-image-setter}`,
so user ends up with `--pattern :`.

Maybe add escapes to the help, or come up with a way to enter patterns on the CL without requiring escapes?